python/dgl/heterograph.py

```diff
@@ -240,9 +235,16 @@ def __init__(self,
             frame.set_initializer(init.zero_initializer)
             self._msg_frames.append(frame)
 
-        self._is_multigraph = multigraph
+        self._is_multigraph = None
+
+    def __getstate__(self):
+        return self._graph, self._ntypes, self._etypes, self._node_frames, self._edge_frames
```

Comment:
We might want to rewrite the pickle implementation for `Frame` also, since it's only a dict and many members in it doesn't need to be involved in pickling

Reply:
Yes but the actual performance doesn't matter too much. See the performance gap in the description to transmitting only two raw pytorch tensor.